french_law/dune

```diff
@@ -0,0 +1,3 @@
+(dirs catala_legifrance)
+
+;; ocaml js python
```

Comment:
?

Reply:
(same as above, this disables most of `french_law`)